WTWSMS/events/ACR_misc_hellenic_flavor.txt

```diff
@@ -5419,6 +5438,7 @@ character_event = {
 		NOT = { has_character_flag = VIET_artemis_sex_question }
 		NOT = { trait = celibate }
 		NOT = { trait = eunuch }
+		supernatural_events_trigger = yes
 	}
 	
 	weight_multiplier = {
```

Comment:
Let me get this straight.

Artemis being a real deity, the player talking directly to Artemis like one would a human, and asking Artemis questions is _NOT_ supernatural?

Absolute hogwash!